### PR TITLE
Add option to flyte-cli for specifying root certificate

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
Adding a switch (`-e` or `--cacert`) that takes a path to a PEM encoded file.  If specified, will be read and used as the root certificate to use in SSL connection with Admin.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
* Adding click option to main flyte-cli command.
* Setting the path in the click context object if specified, None otherwise
* Value is always passed to client constructor. If present, file will be read and used as `root_certificates` value in `grpc.ssl_channel_credentials`

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1888

### CODE REVIEW

File: flytekit/clis/flyte_cli/main.py [hunk 1]
```diff
@@ -306,6 +303,7 @@ def _render_schedule_expr(lp):
 _CONFIG_FLAGS = ["-c", "--config"]
 _PRINCIPAL_FLAGS = ["-r", "--principal"]
 _INSECURE_FLAGS = ["-i", "--insecure"]
+_CERT_FLAGS = ["-e", "--cacert"]
```
Comment (katrogan) [inline flytekit/clis/flyte_cli/main.py:306]: can you remove the "-e" shorthand?

**Review by katrogan — APPROVED** [review]

**Review by katrogan — APPROVED** [review]
